dashboard/factories.py

```diff
@@ -38,9 +38,8 @@ class CachedCertificateFactory(DjangoModelFactory):
         "certificate_type": "verified",
         "grade": randint(60, 100)/100.0,
         "course_id": x.course_run.edx_course_key,
+        "username": x.user.username,
```

Comment:
We will need to be careful about this. Some tests use `UserFactory` but set the edX username to a different name to make sure that we can handle the two being different. Where do we use this username in our code?